classes/chat/ChatManager.php

```diff
@@ -227,7 +227,7 @@ private function fetchPosts(?int $starting_post_id = null, int $max_posts = self
     public function submitPost(string $message) {
         $chat_max_post_length = $this->maxPostLength();
 
-        $message_length = strlen(preg_replace('/[\\n\\r]+/', '', trim($message)));
+        $message_length = strlen(preg_replace('/(\\\\[rn])+/', '', trim($message)));
```

Comment:
What use case is this targeting?

Reply:
New lines are coming in as '\\r\\n'
The original capture group is missing all of them, we need to escape the '\' twice, and then capture 'r' or 'n' any number of times.